llm_sql_prompt/postgres.py

```diff
@@ -135,6 +140,11 @@ def describe_database_and_table(db_url: str, table_names: list[str], all_tables:
         )
 
         for table_name in table_names:
+            # Skip PostgreSQL system tables that might cause access issues
+            if should_skip_table(table_name):
+                print(f"# Skipping table `{table_name}` (PostgreSQL system table)")
```

Comment:
use a logger instead of print

https://github.com/iloveitaly/python-package-template/blob/master/%7B%7Bproject_name_snake_case%7D%7D/__init__.py#L4

follow this pattern